src/main/python/mmlspark/lightgbm/LightGBMRanker.py

```diff
@@ -32,31 +32,39 @@ def saveNativeModel(self, filename, overwrite=True):
         self._java_obj.saveNativeModel(filename, overwrite)
 
     @staticmethod
-    def loadNativeModelFromFile(filename, labelColName="label", featuresColName="features",
-                                predictionColName="prediction"):
+    def loadNativeModelFromFile(filename):
         """
         Load the model from a native LightGBM text file.
         """
         ctx = SparkContext._active_spark_context
         loader = ctx._jvm.com.microsoft.ml.spark.lightgbm.LightGBMRankerModel
-        java_model = loader.loadNativeModelFromFile(filename, labelColName,
-                                                    featuresColName, predictionColName)
+        java_model = loader.loadNativeModelFromFile(filename)
         return JavaParams._from_java(java_model)
 
     @staticmethod
-    def loadNativeModelFromString(model, labelColName="label", featuresColName="features",
-                                  predictionColName="prediction"):
+    def loadNativeModelFromString(model):
         """
         Load the model from a native LightGBM model string.
         """
         ctx = SparkContext._active_spark_context
         loader = ctx._jvm.com.microsoft.ml.spark.lightgbm.LightGBMRankerModel
-        java_model = loader.loadNativeModelFromString(model, labelColName,
-                                                      featuresColName, predictionColName)
+        java_model = loader.loadNativeModelFromString(model)
         return JavaParams._from_java(java_model)
 
     def getFeatureImportances(self, importance_type="split"):
         """
         Get the feature importances as a list.  The importance_type can be "split" or "gain".
         """
         return list(self._java_obj.getFeatureImportances(importance_type))
+
+    def getDenseFeatureShaps(self, vector):
+        """
+        Get the local shap feature importances.
+        """
+        return list(self._java_obj.getFeatureShaps(vector))
```

Comment:
Shouldn’t py4j make java arrays usable as lists on python side? Did this fix an error?

Reply:
this is a method on the model, it wasn't available before.  getFeatureShaps needs to be changed to getDenseFeatureShaps (since it accepts vector on scala side), will fix that with another issue from another PR (shap values for multiclass case) in a follow-up.